e2e-tests/testdata/echo_client/ts/client.ts

```diff
@@ -885,16 +892,20 @@ export class BidiStream<Request, Response> {
         this.connection.close();
     }
 
+    onError(handler: (event: any) => void) {
```

Comment:
Maybe this would be more idiomatic as `.on("error", handler)`

Reply:
do you think we should then also add `.on("errror|message|close", ..)`. Or do we only expose "error" for now, and then keep it open for more things in the future?

Reply:
I've update to `on("error")` now, but didnt add any other events, we can add them later if we want an api for those along with the async iterator. 

But right now there is only one event handler, and maybe it is expected that several handlers can be added with this on-api?

Reply:
I've updated to just proxy `on` to the websockets `addEventListener` (and also added `off`). Let me know what you think